ci/ctest_to_gitlab.sh

```diff
@@ -41,7 +41,7 @@ allocate:
   extends: .daint_alloc
   variables:
     PULL_IMAGE: 'YES'
-    SLURM_TIMELIMIT: '40:00'
+    SLURM_TIMELIMIT: '1:00:00'
```

Comment:
This is ok if the values in line 54 and 70 are updated according to my comments.

Reply:
Updated the other ones.